client/app/events/events.controller.js

```diff
@@ -12,16 +17,20 @@ angular.module('venueApp')
     },
       sectionEvent => {
         $scope.event = sectionEvent;
+      },
+      err => {
+        $scope.err = err;
       });
 
     $scope.goToEventUpload = (id) => {
       $location.path("/student/upload/" + id);
     };
 
     $scope.deleteEvent = (event)=> {
-      if (confirm("This will also delete all sections submissions")) {
+      if (confirm("This will also delete all sections submissions, are you sure you'd like to delete this event?")) {
         SectionEvent.delete({id:event._id}, (response)=>{
           $scope.events.splice($scope.events.indexOf(event), 1);
```

Comment:
I'm getting a "Cannot read property 'splice' of undefined" error and then it doesn't change pages.